docs/interactions/Slash_Commands.md

```diff
@@ -0,0 +1,841 @@
+# Slash Commands
+
+Slash Commands are the new, exciting way to build and interact with apps on Discord.
+
+With Slash Commands, all you have to do is type `/` and you're ready to use your favorite bot. Users can learn everything your bot does and easily find new features as you add them. Validation, error states, and helpful UI walks them through your commands, meaning they can get it right the first time, especially on mobile. You now have one more ally in the fight against your phone's autocorrect.
```

Comment:
We don't have a way for passing non-200 back to the user because we assume it's your API telling Discord that something failed, not necessarily bad user input. It's a bit counterintuitive but we generally expect you to return 200 w/ an error to a user.

Definitely feature request-able and can be re-written as such tomorrow.